contracts/SportMarkets/Parlay/ParlayMarketsAMM.sol

```diff
@@ -553,8 +568,9 @@ contract ParlayMarketsAMM is Initializable, ProxyOwned, ProxyPausable, ProxyReen
         parlayMarketMastercopy = _parlayMarketMastercopy;
     }
 
-    function setParameters(bool _reducedFeesEnabled) external onlyOwner {
+    function setParameters(bool _reducedFeesEnabled, uint _parlaySize) external onlyOwner {
```

Comment:
how was parlaySize set before?
Unrelated, but what does reduceFeesEnabled do?

Reply:
It used the default value at the initialize parlaySize = PARLAY_SIZE

Yea, that is forgotten flag for the reduced fees while buying a position from SportsAMM. 
I will remove it, since it doesn't do anything. I mean remove from the setter.